examples/resources/hpegl_vmaas_load_balancer_monitor/nsx_t_lb_https_monitor.tf

```diff
@@ -0,0 +1,22 @@
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+# HTTPS Monitor
+resource "hpegl_vmaas_load_balancer_monitor" "tf_HTTPS_MONITOR" {
+  lb_id = data.hpegl_vmaas_load_balancer.tf_lb.id 
+  name  =  "tf_HTTPS_MONITOR"       
```

Comment:
if both http and https have the same payload, why are u duplication the schema ?